app/controllers/allocations_controller.rb

```diff
@@ -69,7 +70,7 @@ def recommended_and_nonrecommended_poms_for(prisoner)
     }
 
     pom_response.partition { |pom|
-      pom.position_description.include?(prisoner.current_responsibility)
+      pom.position_description.include?(prisoner.case_owner)
```

Comment:
Is case_owner in this context Custody/Community?

Reply:
Yes that's right - it mirrors the language used here - https://allocation-manager-staging.apps.cloud-platform-live-0.k8s.integration.dsd.io/allocations/edit/G4273GI

I thought by doing this it would be be consistent with NOMIS terminology and make it easier to differentiate against POM responsibility. Just a proposal but happy to reconsider